documentation/projects/proposals/additional_search_views/20230719-implementation_plan_additional_search_views.md

```diff
@@ -414,6 +423,11 @@ not dependent on anything.
 The work on the single result pages (step 7) can be done in parallel with the
 work on the collection pages (step 6), but should follow the previous steps.
 
+The new frontend views can use the existing query parameters (`source`,
+`creator` and `tags` - instead of `tag`) until the API changes are implemented
+since this _will_ return _some_ relevant results, and changing the query
+parameter names is easy.
```

Comment:
Will this apply in production, while this is being developed, or both?

Reply:
Since the additional search views are under a feature flag, this will only be seen on staging. It will apply only while the API changes are being developed.

Reply:
Excellent! Could you add that clarity to the plan?

Reply:
Updated the second step, "2. Add the `additional_search_views` feature flag"